### PR TITLE
Challenge Material Setup

### PR DESCRIPTION
**Pull request by aortizoj15**

### CODE REVIEW

File: components/ChallengeMaterial.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import ChallengeTitleCard from '../components/ChallengeTitleCard'
+import ChallengeQuestionCard from '../components/ChallengeQuestionCard'
+
+const ChallengeMaterial: React.FC = () => {
+  return (
+    <div className="row col-12">
```
Comment (Wolfy64) [inline components/ChallengeMaterial.tsx]: If you removed `col-12` borders will be align like the design
Screen Shot 2020-03-08 at 10 25 51 AM

File: components/ChallengeQuestionCard.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import '../scss/challengeQuestionCard.scss'
+
+type Props = {
+  title: string
+  question: string
+}
+
+const ChallengeQuestionCard: React.FC<Props> = props => {
+  return (
+    <div className="card shadow-sm">
```
[the remark this answers is not in the document]
Comment (Wolfy64) [reply components/ChallengeQuestionCard.tsx]: Why did you remove `shadow-sm` ?
if you look at Zeplin they use a light shadow without border:
Screen Shot 2020-03-08 at 5 16 09 PM
Card should be `<div className="card shadow-sm border-0">`

File: components/ChallengeQuestionCard.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import '../scss/challengeQuestionCard.scss'
+
+type Props = {
+  title: string
+  question: string
+}
+
+const ChallengeQuestionCard: React.FC<Props> = props => {
+  return (
+    <div className="card border-0">
```
Comment (Wolfy64) [inline components/ChallengeQuestionCard.tsx]: Why did you remove `shadow-sm` ?
if you look at Zeplin they use a light shadow without border:
Screen Shot 2020-03-08 at 5 16 09 PM
Card should be `<div className="card shadow-sm border-0">`

File: components/ChallengeQuestionCard.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+type Props = {
+  title: string
+  question: string
+}
+
+const ChallengeQuestionCard: React.FC<Props> = props => {
+  return (
+    <div className="card shadow-sm border-0">
+      <div className="card-body">
+        <h1 className="challenge-question-card__title">{props.title}</h1>
```
Comment (songz) [inline components/ChallengeQuestionCard.tsx]: could this just all be `{content}` we will use markdown to generate the HTML content.

File: components/ChallengeMaterial.tsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import ChallengeTitleCard from '../components/ChallengeTitleCard'
+import ChallengeQuestionCard from '../components/ChallengeQuestionCard'
+
+const ChallengeMaterial: React.FC = () => {
+  return (
+    <div className="row">
+      <div className="col-4">
+        <ChallengeTitleCard challengeNum={1} title="Less Than or Equal to 5" />
```
Comment (songz) [inline components/ChallengeMaterial.tsx]: I think `components` folder is getting cluttered with too many files. Wanna put `ChallengeTitleCart` and `ChallengeQuestionCard` in this file since they are not used anywhere else?
